Assets/MRTK.ThirdParty/MRTK-Quest/Scripts/Config/MRTKOculusConfig.cs

```diff
@@ -0,0 +1,297 @@
+        public bool UseCustomHandMaterial => useCustomHandMaterial;
+
+        [SerializeField]
+        [Tooltip("Custom hand material to use for hand tracking hand mesh.")]
+        private Material customHandMaterial = null;
+
+        /// <summary>
+        /// Event triggered when the custom material for hand mesh is updated.
+        /// </summary>
+        public static System.Action OnCustomHandMaterialUpdate;
+
+        /// <summary>
+        /// Custom hand material to use for hand tracking hand mesh.
+        /// </summary>
+        public Material CustomHandMaterial
+        {
+            get => customHandMaterial;
+
+            set
+            {
+                customHandMaterial = value;
+                OnCustomHandMaterialUpdate?.Invoke();
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("If true, will update material pinch strength using OVR Values.")]
+        private bool updateMaterialPinchStrengthValue = true;
+
+        /// <summary>
+        /// If true, will update material pinch strength using OVR Values.
+        /// </summary>
+        public bool UpdateMaterialPinchStrengthValue => UseCustomHandMaterial && updateMaterialPinchStrengthValue;
+
+        [SerializeField]
+        [Tooltip("Property in custom material used to visualize pinch strength.")]
+        private string pinchStrengthMaterialProperty = "_PressIntensity";
+
+        /// <summary>
+        /// Property in custom material used to visualize pinch strength.
+        /// </summary>
+        public string PinchStrengthMaterialProperty => pinchStrengthMaterialProperty;
+
+#if OCULUSINTEGRATION_PRESENT
+        [Header("Hand Tracking Configuration")]
+        [SerializeField]
+        [Tooltip("Setting this to low means hands will continue to track with low confidence.")]
+        private OVRHand.TrackingConfidence _minimumHandConfidence = OVRHand.TrackingConfidence.Low;
+
+        /// <summary>
+        /// Setting this to low means hands will continue to track with low confidence.
+        /// </summary>
+        public OVRHand.TrackingConfidence MinimumHandConfidence
+        {
+            get => _minimumHandConfidence;
+            set => _minimumHandConfidence = value;
+        }
+
+        /// <summary>
+        /// Current tracking confidence of left hand. Value managed by OculusQuestHand.cs.
+        /// </summary>
+        public OVRHand.TrackingConfidence CurrentLeftHandTrackingConfidence { get; set; }
+
+        /// <summary>
+        /// Current tracking confidence of right hand. Value managed by OculusQuestHand.cs.
+        /// </summary>
+        public OVRHand.TrackingConfidence CurrentRightHandTrackingConfidence { get; set; }
+#endif
+
+        [SerializeField]
+        [Range(0f, 5f)]
+        [Tooltip("Time after which low confidence is considered unreliable, and tracking is set to false. Setting this to 0 means low-confidence is always acceptable.")]
+        private float _lowConfidenceTimeThreshold = 0.2f;
```

Comment:
_lowConfidenceTimeThreshold  -> lowConfidenceTimeThreshold (here and below).

Same with other variables that prefix with _ (which I'm guessing either pulled from some other old C++ style or other style guideline)